docs/OCP_CI_Tutorials/Scenarios/Reporting_Guide.md

```diff
@@ -0,0 +1,128 @@
+# Reporting Guide<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+
+- [Introduction](#introduction)
+- [TestGrid](#testgrid)
+  - [TestGrid Configuration Example and Description](#testgrid-configuration-example-and-description)
+  - [How to Create a New Dashboard](#how-to-create-a-new-dashboard)
+  - [How to Add a Job to a Dashboard](#how-to-add-a-job-to-a-dashboard)
+  - [Pull Request Process](#pull-request-process)
+- [Report Portal](#report-portal)
+- [Slack](#slack)
+
+
+## Introduction
+
+Because OpenShift CI does not have reporting built into it's logic, we must configure and maintain our reporting step. This process may change as we evolve our processes in OpenShift CI, but we have tried to make this step as easy as possible. Please see each section below to see how to report your scenarios to each of the platforms.
```

Comment:
```suggestion
Because OpenShift CI does not have reporting built into its logic, we must configure and maintain our reporting step. This process may change as we evolve our processes in OpenShift CI, but we have tried to make this step as easy as possible. Please see each section below to see how to report your scenarios to each of the platforms.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)